build-tools/setup-hooks.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
```

Comment:
have you considered using this instead?

```suggestion
#!/usr/bin/env bash
```

Reply:
I would be fine either way. If there's a preference - do it.